src/components/dialogs/field-hook.js

```diff
@@ -250,9 +254,30 @@ export const useNameField = ({
             name.replace(/ /g, '') !== '' &&
             !error &&
             !checking,
+        setName,
     ];
 };
 
+export const usePrefillNameField = ({ nameRef, selectedFile, setValue }) => {
+    useEffect(() => {
+        if (
+            nameRef !== undefined &&
+            nameRef.current.trim().length === 0 &&
+            selectedFile != null
+        ) {
+            setValue(
+                selectedFile.name.substr(0, selectedFile.name.indexOf('.'))
+            );
+        } else if (
+            selectedFile == null &&
+            setValue != null &&
+            setValue !== undefined
+        ) {
```

Comment:
use === and !==

Reply:
But your test on setValue should be upper nope ?
setValue is also used in the other branch of the if

Reply:
Done !